src/gui/widgets/scroll_text.cpp

```diff
@@ -161,6 +163,8 @@ void scroll_text::place(const point& origin, const point& size) {
 			scroll_vertical_scrollbar(scrollbar_base::BEGIN);
 		}
 	}
+
+	set_max_size(widget->get_config_default_size());
```

Comment:
`widget` can be null here (or if it can't then the condition block doesn't need to be conditional)

Reply:
I'll check that.

Reply:
Moved `set_max_size` inside.